Make.inc

```diff
@@ -103,6 +104,12 @@ ifeq ($(OS), WINNT)
   shlibdir = $(libdir)
 endif
 
+ifeq ($(OS), FreeBSD)
```

Comment:
You should move this code above, and use it instead of the unconditional definition. Also, use `$(libdir)` as I said above, since people may override it.